### PR TITLE
Oracle Support WIP

### PR DESCRIPTION
**Pull request by chook**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ npm-debug.log
 coverage*
 .tern-port
 v8.log
+package-lock.json
```
Comment (davidlevanon) [inline .gitignore]: package-lock.json should not be ignored 

https://docs.npmjs.com/files/package-lock.json

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
   - "4"
+  - "6"
+  - "8"
+  - "10"
```
Comment (davidlevanon) [inline .travis.yml]: dynalite doesn't work with node 10

File: index.js [hunk 3]
```diff
@@ -13,6 +13,8 @@ var http = require('http'),
 		dynaliteLogger = require('./logger'),
 		stats = require('./stats');
 
+require('es6-promise').polyfill();
```
Comment (davidlevanon) [inline index.js]: is this required?

**Review by davidlevanon — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (chook) [reply .gitignore]: did i do that? :D then revert

File: .travis.yml (hunk 2 above)
Comment (chook) [reply .travis.yml]: i have no idea how this happened .. but who cares about travis.yml?

File: index.js (hunk 3 above)
Comment (chook) [reply index.js]: it doesn't work otherwise on my Mac.

**Pull request merged**